docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: '3.9'
+
+
+
+services:
+  ficai-db:
+    container_name: ficai-db
+    image: postgres:14-alpine
+    restart: unless-stopped
+    security_opt:
+      - no-new-privileges:true
+    expose:
+      - 5432
+    environment:
+      POSTGRES_DB: $FICAI_DB_DATABASE
+      POSTGRES_USER: $FICAI_DB_USERNAME
+      POSTGRES_PASSWORD: $FICAI_DB_PASSWORD
+    volumes:
+      - './schema.sql:/docker-entrypoint-initdb.d/schema.sql'
+      - 'db_data:/var/lib/postgresql/data/'
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U $FICAI_DB_USERNAME -d $FICAI_DB_DATABASE" ]
+      interval: 10s
+      timeout: 5s
+      retries: 10
+      start_period: 10s
+
+  ficai-signals:
+    container_name: ficai-signals
+    build: .
+    restart: unless-stopped
+    security_opt:
+      - no-new-privileges:true
+    depends_on:
+      - ficai-db
+    ports:
+      - '8080:8080'
```

Comment:
This also implies that `FICAI_LISTEN` is set to `0.0.0.0:8080`, so that probably shouldn't be configurable via `.env` either.